### DISCUSSION TITLE
No documentation on using multiple attribute for select() form helper

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1432] [General]
I am surprised that the API does not even mention the "multiple" attribute available for the select() form tag helper. I can't find by searching any mention of it in the Guides.

Please update the Guides and API and provide several examples of how to use it for the select and for saving multiple values when saving multiple selections.

**Comment by chapmandu** [comment]
This is a native HTML input attribute, no different to class or style. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple

**Comment by zainforbjs** [comment]
@cfcoder I'm gonna close this one as well as this is answered over created issue #1433
